tests/clients/test_experiment.py

```diff
@@ -121,6 +128,11 @@
 }
 
 
+def test_tag_schema():
```

Comment:
We should probably test the `.dump()` on this as well as it's being used for serialisation.